ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__multi-nightly.yaml

```diff
@@ -349,6 +349,244 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb-subnets-day2
+- as: aws-ipi-lb-role-isolate-node-mini-perm-arm-f14
```

Comment:
Can we merge role setting testing to the existing ci jobs ?

Reply:
Updated.